AssemblyInfo.Shared.cs

```diff
@@ -22,9 +22,9 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("6.3.0")]
-[assembly: AssemblyFileVersion("6.3.0.0")]
-[assembly: AssemblyInformationalVersion("Version:6.3.0.0 Branch:not-set Sha1:not-set")]
+[assembly: AssemblyVersion("7.0.0")]
```

Comment:
The actual breaking change was this:
https://github.com/SonarSource/sonar-scanner-msbuild/pull/1967#discussion_r1645881887
I added a card to the Kanban, so we do not forget about it in the release notes.